app/models/proxy.rb

```diff
@@ -537,6 +537,9 @@ class PoliciesConfig
     include Enumerable
 
     delegate :each, :to_json, :as_json, to: :policies_config
+
+    # this alias is needed because of
+    # https://github.com/rsim/oracle-enhanced/issues/2200
```

Comment:
Same not needed here.

Reply:
Oh, this is for the PR in progress I have. I will remove it. Or you mean we should not have it at all in any PR?